scripts/query.py

```diff
@@ -0,0 +1,25 @@
+from qdrant_client import QdrantClient
+
+from openai import OpenAI
+
+
+def main():
+    client = QdrantClient("0.0.0.0", port=6333)
+    openai_client = OpenAI()
+
+    model = "text-embedding-ada-002"
+    prompt = ""
+    embeddings = openai_client.embeddings.create(
+        input = prompt,
+        model=model
+    ).data[0].embedding
+    results = client.search(
+        collection_name="1000",
```

Comment:
I think this was an accident, the collection name shouldn't be 1000 here.

Reply:
I created a commit to fix it.